device/esp_tinyusb/Kconfig

```
@@ -185,6 +185,36 @@ menu "TinyUSB Stack"
             default "/data"
             help
                 MSC Mount Path of storage.
+
+        menu "TinyUSB FAT Format Options"
+            choice TINY_USB_FAT_FORMAT_TYPE
+               prompt "FatFS Format Type"
+                default TINY_USB_FAT_FORMAT_ANY
+                help
+                    Select the FAT filesystem type used when formatting storage.
+
+            config TINY_USB_FAT_FORMAT_ANY
```

```suggestion
            config TINYUSB_FAT_FORMAT_ANY
```

Could you please update the config names, to it matches with already used style? Same comment for the rest of the configs as well.

---

@erwinschrodinger1 thank you for the update but you need to update all the config names in this file and then also in device/esp_tinyusb/tusb_msc_storage.c -> without the change in .c file the Kconfig options will not take effect

---

Updated in newest commit.